### DISCUSSION TITLE
Wazirx

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#2058] [General]
What should be the best option according to you guys, when a exchange falls !!
1)Sell your crypto and convert into fiat currency and wait until exchange allows to withdraw fiat
2)wait until the exchange allows to withdraw the crypto into other wallet
3)wait for prices to came up then sell the crypto and withdraw fiat

What should be the best practice?
*Just for knowledge, not a financial advice !!

**Comment by EngrPips** [comment]
I am not even sure what should be best practice, I am looking forward to what people think.
